app/javascript/custom/recipes.js

```diff
@@ -0,0 +1,81 @@
+const fileInput = document.querySelector('.file input[type=file]');
+const ingredientAdder = document.querySelector('#ingredient_adder');
+const applianceAdder = document.querySelector('#appliance_adder');
+
+var selectedIngredients = [];
```

Comment:
It cannot be a const because it has to be updated and it has to be a global var because it has to persist its contents through the whole process of creating a recipe

Reply:
You sure it can't be a `let`?

Reply:
Pretty sure you are right. changed vars into lets in the latest commit :3